tutorial/practicals/LFRic/building_code/2_built_ins/README.md

```diff
@@ -370,7 +370,7 @@ The metadata is very similar to a user-defined LFRic kernel metadata
 with one major difference: built-ins are called from the [PSy-layer](
 ../background/LFRic_structure.md#psy-layer) loops over degrees of
 freedom (*DoFs*) of fields in the built-in, hence metadata identifier
-for this way of looping, `DOFS`. This means that the fields passed to
+for this way of looping, `DOF`. This means that the fields passed to
```

Comment:
"...) from within loops over the degrees of freedom (*DoFs*) of the field arguments. Hence the `operates_on` metadata for this type of kernel has the `DOF` value. All fields passed to..."

Reply:
Thanks for the suggestion, done.